docs/rdbms/naming-convention.md

```diff
@@ -0,0 +1,134 @@
+---
+id: rdbms-naming-convention
+title: RDBMS Naming Convention
+sidebar_label: Naming Convention
+---
+
+#### The following convention should be followed for naming RDBMS:
+
+* Use `snake_case` with lowercase while naming.
+* Use uppercase for SQL keywords and built-in functions.
+* Avoid use of dots, spaces, or dashes in database, schema, table, or column names.
+
```

Comment:
* When naming objects, balance the objective of keeping names short and easy to use with the objective of making names as descriptive as possible. When in doubt, choose the more descriptive name, because the objects in the database may be used by many people over a period of time.
For example use `payment_due_date` instead of `pmdd`